peer_mentoring/account_management/models.py

```diff
@@ -31,19 +26,42 @@ class CustomUser(AbstractUser):
     first_name = models.CharField(max_length=150, blank=False, help_text="Required. 150 characters or less.", validators=[MaxLengthValidator(150), MinLengthValidator(1)], error_messages={'blank':'A first name is required.'})
     last_name = models.CharField(max_length=150, blank=False, help_text="Required. 150 characters or less.", validators=[MaxLengthValidator(150), MinLengthValidator(1)], error_messages={'blank':'A last name is required.'})
     dob = models.DateField(blank=False, help_text="Required, please enter a date", )
-    username = models.CharField(max_length=150, unique=True, help_text="Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.",
+    username = models.CharField(max_length=150, blank=False, unique=True, help_text="Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.",
                                 validators=[username_validator, MaxLengthValidator(150),MinLengthValidator(1)], error_messages={"unique":"A user with that username already exists."})
     email = models.EmailField(unique=True, blank=False, validators=[EmailValidator()] )
     password = models.CharField(blank=False, max_length=127)
     date_joined = models.DateTimeField(default=datetime.now)
 
+    def __str__(self):
+        return self.first_name, self.last_name, self.dob, self.username, self.email, self.date_joined
```

Comment:
this seems a tuple? should be a string

Reply:
Should we do __repr__ to return more information?

Reply:
repr is to show how an object gets created, we generally know that for django models so I would leave it off here

FYI difference str vs repr:

<img width="736" alt="image" src="https://user-images.githubusercontent.com/387927/221633948-54447361-25a2-4664-9679-237444fb8451.png">

Reply:
Okay, thank you.

Reply:
So what would be best to return multiple variables? Variable are readily accessible anyways, right?

Reply:
Sorry I don't follow, can you clarify please?

`__str__` is supposed to return a single string, not multiple values (tuple) which seems to be the case how you have it now. 

```
>>> def func():
...     return 1, 2
...
>>> def func2():
...     return "1, 2"
...
>>> func()
(1, 2)
>>> func2()
'1, 2'
>>> type(func())
<class 'tuple'>
>>> type(func2())
<class 'str'>
```

Reply:
I meant that I can always access the model fields through a queryset anyways if I need to access the data to render anything on the templates. Is that right, or am I wrong in how I did that before?

Reply:
Yes true, but see this as a quick summary of the object when you inspect it in the shell or debugger ...

Reply:
I see the point now